pkg/antctl/raw/multicluster/deploy/leader_cluster.go

```diff
@@ -0,0 +1,83 @@
+// Copyright 2022 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package deploy
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+)
+
+type leaderClusterOptions struct {
+	namespace     string
+	filename      string
+	antreaVersion string
+}
+
+var leaderClusterOpts *leaderClusterOptions
+
+var leaderClusterExamples = strings.Trim(`
+Create the leader cluster CRDs and deploy the "antrea-mc-controller" Deployment in Namespace antrea-mcs.
+$ antctl mc deploy leader-cluster --antrea-version v1.6.0 -n antrea-mcs
+Create the leader cluster CRDs and deploy the "antrea-mc-controller" Deployment using pre-downloaded manifest
+$ antctl mc deploy leader-cluster -f manifest.yml
+
+The following CRDs will be created:
+- CRDs: ClusterClaim, ClusterSet, MemberClusterAnnounce, ResourceExportFilter, ResourceExport, ResourceImport, ResourceImportFilter, ServiceExport, ServiceImport
+`, "\n")
+
+func (o *leaderClusterOptions) validateAndComplete() error {
+	if o.filename != "" {
+		return nil
+	}
+	if o.namespace == "" {
+		return fmt.Errorf("the Namespace cannot be empty")
+	}
+	if o.antreaVersion == "" {
+		o.antreaVersion = "latest"
+	}
+
+	return nil
+}
+
+func NewLeaderClusterCmd() *cobra.Command {
+	command := &cobra.Command{
+		Use:     "leader-cluster",
+		Args:    cobra.MaximumNArgs(0),
+		Short:   "Deploy Antrea Multi-cluster to a leader cluster",
+		Long:    "Deploy Antrea Multi-cluster to a leader cluster in a Namespace",
+		Example: leaderClusterExamples,
+		RunE:    leaderClusterRunE,
+	}
+	o := &leaderClusterOptions{}
+	leaderClusterOpts = o
+	command.Flags().StringVarP(&o.namespace, "namespace", "n", "", "Namespace to deploy Antrea Multi-cluster")
+	command.Flags().StringVarP(&o.antreaVersion, "antrea-version", "", "",
+		"version of Antrea Multi-cluster to deploy. If not set, the latest version from Antrea main will be used."+
+			"When manifest-file is not specified, antctl will download and apply the specified version of Antrea Multi-cluster deployment manifest;"+
```

Comment:
Probably change to "When manifest-file is not provided, the Antrea Multi-cluster deployment manifest of the specified version will be downloaded and applied; " to be consistent with other sentences.

Add a space after ";"

Reply:
done.  Did the same in member_cluster.